lib_nn/api/vpu_sim.h

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 XMOS LIMITED. This Software is subject to the terms of the
+// XMOS Public License: Version 1
+#pragma once
```

Comment:
It might be better to use include guards instead of pragma once? For consistency and portability.